DigitalLearningSolutions.Web/Views/RegisterDelegateByCentre/PersonalInformation.cshtml

```diff
@@ -0,0 +1,70 @@
+@using DigitalLearningSolutions.Web.ViewModels.Register
+@model PersonalInformationViewModel
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = errorHasOccurred ? "Error: Register Delegate" : "Register Delegate";
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <form class="nhsuk-u-margin-bottom-8" method="post" asp-action="PersonalInformation">
+      @if (errorHasOccurred) {
+        <vc:error-summary order-of-property-names="@(new[] { nameof(Model.Centre), nameof(Model.FirstName), nameof(Model.LastName), nameof(Model.Email), nameof(Model.Alias) })" />

```

Comment:
As the centre is fixed for these pages, I think the "nameof(Model.Centre)" can be safely removed from this error summary.